subsys/net/lib/lwm2m/Kconfig

```diff
@@ -122,6 +122,7 @@ config LWM2M_FIRMWARE_UPDATE_PULL_SUPPORT
 	bool "Firmware Update object pull support"
 	default y
 	depends on LWM2M_FIRMWARE_UPDATE_OBJ_SUPPORT
+	select HTTP_PARSER_URL_ONLY
```

Comment:
You need to add "select HTTP_PARSER" here too as when doing "select" in Kconfig, it will not recursively select other options.